mesos_slave/tests/test_integration_e2e.py

```diff
@@ -12,8 +10,6 @@
 from .common import CHECK_NAME
 
 
-# Linux only: https://github.com/docker/for-mac/issues/1031
-@pytest.mark.skipif(platform.system() != 'Linux', reason='Only runs on Unix systems')
```

Comment:
Does it run on windows?

Reply:
Seems it does not. Checking if that's also the case for mesos master: https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=1591&view=results

If so, I will find a way to make it work, or add a `skipif` condition to not run on windows.

Reply:
Just add the skipif windows

Reply:
Yes, I can do that. Not sure if it worth investing why it's not working on windows right now.

Reply:
@hithwen Done